openslides_backend/action/mixins/import_mixins.py

```diff
@@ -0,0 +1,129 @@
+from enum import Enum
+from time import time
+from typing import Any, Callable, Dict, List, Optional
+
+from ...shared.interfaces.event import Event, EventType
+from ...shared.interfaces.write_request import WriteRequest
+from ...shared.patterns import fqid_from_collection_and_id
+from ..action import Action
+from ..util.typing import ActionData, ActionResultElement
+
+
+class ImportStatus(str, Enum):
+    ERROR = "error"
+    NEW = "new"
+    WARNING = "warning"
+    DONE = "done"
```

Comment:
Add `generated` here and use that as well instead of the literal string.

Reply:
Done.